mobile/.env.example

```
@@ -5,3 +5,26 @@ EXPO_PUBLIC_SOROBAN_RPC_URL=https://soroban-testnet.stellar.org
+# Master switch. Set EXPO_PUBLIC_PINNING_ENABLED=false to disable pinning
+# entirely (dev/test); leave unset (or true) to enable.
+EXPO_PUBLIC_PINNING_ENABLED=false
+
+# JSON mapping of hostname -> array of RFC 7469 SPKI pins (base64 SHA-256 of
+# the DER-encoded SubjectPublicKeyInfo). At least one pin must match the peer
+# certificate. Ship >= 2 pins (current + next) so rotation never locks users out.
+# Generate with:
+#   openssl s_client -connect api.example.com:443 -servername api.example.com \
+#     </dev/null 2>/dev/null | openssl x509 -pubkey -noout | \
+#     openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | openssl base64
+EXPO_PUBLIC_PINNING_POLICY={"api.example.com":["sha256/<base64>","sha256/<base64>"]}
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)